src/browser/components/FileDrop/FileDrop.jsx

```diff
@@ -22,15 +22,20 @@ import React, { useState } from 'react'
 import { connect } from 'react-redux'
 import { withBus } from 'react-suber'
 import SVGInline from 'react-svg-inline'
+import { every } from 'lodash-es'
 
 import * as editor from 'shared/modules/editor/editorDuck'
-import { addFavorite } from 'shared/modules/favorites/favoritesDuck'
+import {
+  addFavorite,
+  addManyFavorites
+} from '../../../shared/modules/user-favorites/user-favorites.duck'
```

Comment:
Rather than traversing the file tree, could you use the `shared` alias here?

Reply:
I can :) Generally I find these webpack aliases to be a bit of an anti-pattern but since this is already a norm in browser I'll update them